operators/sonataflow-operator/10.0.0/manifests/sonataflow-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,886 @@
+                - create
+                - update
+                - delete
+                - deletecollection
+                - patch
+                - watch
+            - apiGroups:
+                - build.openshift.io
+              resources:
+                - buildconfigs
+                - builds
+              verbs:
+                - create
+                - delete
+                - deletecollection
+                - get
+                - list
+                - patch
+                - update
+                - watch
+            - apiGroups:
+                - build.openshift.io
+              resources:
+                - buildconfigs/finalizers
+                - builds/finalizers
+              verbs:
+                - get
+                - list
+                - create
+                - update
+                - delete
+                - deletecollection
+                - patch
+                - watch
+            - apiGroups:
+                - build.openshift.io
+              resources:
+                - buildconfigs/instantiatebinary
+              verbs:
+                - create
+            - apiGroups:
+                - authentication.k8s.io
+              resources:
+                - tokenreviews
+              verbs:
+                - create
+            - apiGroups:
+                - authorization.k8s.io
+              resources:
+                - subjectaccessreviews
+              verbs:
+                - create
+            - apiGroups:
+                - apps
+              resources:
+                - statefulset
+                - statefulsets
+              verbs:
+                - get
+                - list
+                - watch
+            - apiGroups:
+                - networking.k8s.io
+              resources:
+                - ingress
+                - ingresses
+              verbs:
+                - get
+                - list
+                - watch
+            - apiGroups:
+                - serving.knative.dev
+              resources:
+                - service
+                - services
+              verbs:
+                - get
+                - list
+                - watch
+            - apiGroups:
+                - eventing.knative.dev
+              resources:
+                - broker
+                - brokers
+              verbs:
+                - get
+                - list
+                - watch
+            - apiGroups:
+                - apps.openshift.io
+              resources:
+                - deploymentconfigs
+              verbs:
+                - get
+                - list
+                - watch
+            - apiGroups:
+                - route.openshift.io
+              resources:
+                - routes
+              verbs:
+                - get
+                - list
+                - watch
+          serviceAccountName: sonataflow-operator-controller-manager
+      deployments:
+        - label:
```

Comment:
Remove this label from here. This control-plane label is not correctly placed. The tests are failing due to the incorrect format here.